source/data/tier1_data_architecture.rst

```diff
@@ -9,18 +9,14 @@ The VSC Tier-1 Data component is based on the open source software iRODS. The fo
 .. image:: architecture/general_overview.png
 
 The current deployment is based in a unique iRODS zone (“kuleuven_tier1_pilot”) with a single iCAT database configured on High Availability. There are three distributed storage resources: 2 POSIX based systems and 1 Ceph Object Storage system). 
-One of the POSIX systems and the Ceph based storage are physically installed at the KU Leuven Heverlee datacenter while the other POSIX system is installed at the KU Leuven datacenter located in Leuven center and act as a replica of the POSIX system located in Heverlee. 
+One of the POSIX systems and the Ceph based storage are physically installed at the KU Leuven Heverlee datacenter while the other POSIX system is installed at the KU Leuven datacenter located in Leuven center and acts as a replica of the POSIX system located in Heverlee. 
 
 That means that for every object registered in the Data platform 2 copies will be automatically stored (one in Heverlee and one Leuven). 
 
 The third storage system will be used for the moment to store a third copy of some selected data objects. 
```

Comment:
That's not true anymore. I suggest to remove this line

Reply:
Ok I will discuss the new situation with you.